### PR TITLE
wgsl: Implement scalar/vector AbstractFloat addition execution tests

### PR DESCRIPTION
**Pull request by zoddicus**
Matrix addition will be covered in future PRs.

Issue #1626

<hr>

**Requirements for PR author:**

- [x] All missing test coverage is tracked with "TODO" or `.unimplemented()`.
- [x] New helpers are `/** documented */` and new helper files are found in `helper_index.txt`.
- [x] Test behaves as expected in a WebGPU implementation. (If not passing, explain above.)

**Requirements for [reviewer sign-off](https://github.com/gpuweb/cts/blob/main/docs/reviews.md):**

- [ ] Tests are properly located in the test tree.
- [ ] [Test descriptions](https://github.com/gpuweb/cts/blob/main/docs/intro/plans.md) allow a reader to "read only the test plans and evaluate coverage completeness", and accurately reflect the test code.
- [ ] Tests provide complete coverage (including validation control cases). **Missing coverage MUST be covered by TODOs.**
- [ ] Helpers and types promote readability and maintainability.

When landing this PR, be sure to make any necessary issue status updates.

### CODE REVIEW

File: src/webgpu/util/math.ts [hunk 1]
```diff
@@ -1728,6 +1728,38 @@ export function vectorF64Range(dim: number): number[][] {
...
+  4: sparseF64Range().map((f, idx) => [
+    idx % 4 === 0 ? f : idx,
+    idx % 4 === 1 ? f : -idx,
+    idx % 4 === 2 ? f : idx,
+    idx % 4 === 3 ? f : -idx,
+  ]),
+};
+
+/**
+ * Minimal set of vectors, indexed by dimension, that contain interesting f16
```
Comment (jzm-intel) [inline src/webgpu/util/math.ts:1747]:
```suggestion
 * Minimal set of vectors, indexed by dimension, that contain interesting f64
```

**Review by jzm-intel — APPROVED** [review]
Looks good to me, with tiny typo.

File: src/webgpu/util/math.ts (hunk 1 above)
Comment (zoddicus) [reply src/webgpu/util/math.ts]: Done

File: src/unittests/floating_point.spec.ts [hunk 2]
```diff
@@ -3663,12 +3663,27 @@ const kAdditionInterval64BitsNormalCases = {
...
+    // 0.1 isn't exactly representable in f64, but will be quantized to an
+    // exact value when storing to a 'number' (0x3FB999999999999A).
+    // This is why below the expectations are not intervals.
+    { input: [0.1, 0], expected: [0.1] },
+    { input: [0, 0.1], expected: [0.1] },
+    { input: [-0.1, 0], expected: [-0.1] },
+    { input: [0, -0.1], expected: [-0.1] },
+    // f64 0x3FB999999999999A+0x3FB999999999999A = 0x3FC999999999999A
+    { input: [0.1, 0.1], expected: [reinterpretU64AsF64(0x3FC999999999999An)] },  // ~0.2
+    // f64 0xBFB999999999999A+0x3FB999999999999A = 0xBFC999999999999A
```
[the remark this answers is not in the document]
Comment (zoddicus) [reply src/unittests/floating_point.spec.ts]: Done

**Pull request merged**